contracts/polls/PollManager.sol

```diff
@@ -180,7 +206,7 @@ contract PollManager is Controlled {
 
         uint totalBallots = 0;
         for(uint8 i = 0; i < _ballots.length; i++){
-            totalBallots += _ballots[i];
+            totalBallots += totalBallots.add(_ballots[i]);
```

Comment:
I think the `+=` should be changed for `=`

Reply:
Yikes. Fixed.